test/support/test-records.json

```diff
@@ -119,5 +119,15 @@
       "cause": null,
       "isDupliciate": false
     }
+  },
+  {
+    "type": "pixel",
+    "destination": "dataset2.pixels",
```

Comment:
These actually get hardcoded into a select field in [pixel.prx.org](https://github.com/PRX/pixel.prx.org/blob/master/lib/pages.js#L34).  And since pixel tracks are signed over there, should be able to trust them.